0016-pre-shared-key/0016-pre-shared-key.md

```diff
@@ -14,6 +14,28 @@ An advantage of PSK is that the pre-shared secret key only needs to be shared on
 
 A disadvantage of PSK is that it is repudiable. Although the sender does get cryptographic proof that the recipient received the payment, that proof cannot be used to convince 3rd parties that the sender did indeed send the funds, because the sender could have generated the fulfillment themselves. However, simple proof for the sender that the recipient got the funds is sufficient in many applications. The [Interledger Payment Request](https://github.com/interledger/rfcs/blob/master/0011-interledger-payment-request/0011-interledger-payment-request.md) transport protocol should be used instead in cases where non-repudiable proof is required.
 
+## Importance of PSK in ensuring network health
+
+When a sender or connector asks  the next connector to forward a payment to the receiver, they need to specify, among other things, the end-to-end data which may convince the receiver to fulfill the
+condition, and the source amount which will reward that next connector.
+
+PSK enables the safe delivery of the end-to-end data, which in turn allows the receiver to check whether the on-ledger destination amount is high enough.
+If the on-ledger destination amount is too low, that means either the sender, or one of the connectors along the path, used a transfer amount that was too low (i.e., is effectively trying to steal
+a little bit).
+
+If the destination amount is too low, even if just by a few percent, the receiver should reject the payment. If this happens, the sender should retry the payment, using a different route.
+This way, eventually, connectors that try to steal money will lose traffic, which is how the Interledger protocol stack ensures the network stays healthy.
+It is PSK's end-to-end data transportation that makes this continuous price competition possible, so that is one of the main advantages of PSK.
+
+The destination amount can either be public or secret. If the destination amount is public, each connector along the path can make sure they pay a fair but sufficient outgoing transfer amount.
```

Comment:
Right now the amount is always in the packet and public, and that isn't a decision on the part of PSK. The question in https://github.com/interledger/rfcs/issues/312 is whether the amount should be looked at by connectors or whether it should be in the packet at all. If we decided to take it out of the ILP packet, we would update PSK so that it always has the amount in the encrypted data.

Reply:
> update PSK so that it always has the amount in the encrypted data.

yes, that's what I was referring to with 'making the amount secret'